migrations/versions/76c949b1235f_.py

```diff
@@ -33,7 +33,8 @@ def upgrade():
     op.alter_column('events_version', 'pay_onsite', new_column_name='can_pay_onsite')
     op.alter_column('events_version', 'tax_allow', new_column_name='is_tax_enabled')
     op.alter_column('events_version', 'ticket_include', new_column_name='is_ticketing_enabled')
-    op.alter_column('events_version', 'has_session_speakers', new_column_name='has_sessions_speakers')
+    op.alter_column('events_version', 'has_session_speakers',
```

Comment:
@SaptakS 
This should have been 
`op.alter_column('events', 'has_sessions_speakers', new_column_name='is_sessions_speakers_enabled')`
made a fix in [PR#3846](https://github.com/fossasia/open-event-orga-server/pull/3846)